e2e/commands/patient-operations.ts

```diff
@@ -59,3 +59,8 @@ export const deletePatient = async (api: APIRequestContext, uuid: string) => {
   const response = await api.delete(`patient/${uuid}`);
   await expect(response.ok()).toBeTruthy();
 };
+
+export function getPatientIdentifierStr(patient: Patient) {
+  const openmrsIdentifier = patient.identifiers[0].display.split('=')[1].trim();
+  return openmrsIdentifier;
```

Comment:
```suggestion
  return patient.identifiers[0].display.split('=')[1].trim();
```

Reply:
Or, if this is a "real" FHIR object and not just constructed from the REST API:

```suggestion
  return patient.identifiers[0].identifier;
```

Maybe:

```ts
const fhirIdentifier = patient.identifiers[0];
return fhirIdentifier.identitifer ?? fhirIdentifier.display.split('=')[1].trim();
```